### ISSUE TITLE
GameCrash

### ISSUE THREAD

**Issue by DavideAntoniazzi** [CorgiTaco-MC/CorgiLib#41]
this file make my modpack not working

**Comment by winsrp** [comment]
same here, no error message, it will just crash minecraft, using minecraft 1.20.1, corgilib-forge-4.0.2.0

**Comment by xg787** [comment]
downgrade to 4.0.1.3

**Comment by rmyj** [comment]
latest release break modpack - that was a hard problem to find - no error message in my minecraft or forge log file

**Comment by nanonestor** [comment]
It appears that this mod may including Forge's 'nightconfig' system which breaks forge loading correctly.  Not sure of the details

```
  "jars": [
    {
      "identifier": {
        "group": "com.electronwill.night-config",
        "artifact": "core"
      },
      "version": {
        "range": "[3.6.7,)",
        "artifactVersion": "3.6.7"
      },
      "path": "META-INF/jars/core-3.6.7.jar"
    },
```

edit - yeah upon inspection the actually built mod file 4.0.2.0 ships META-INF\jars\core-3.6.7.jar inside, which causes the conflict and issue.

**Comment by DragonBlak** [comment]
just downgrade to 4.0.1.3 for now

**Comment by nanonestor** [comment]
Seeing version 4.0.2.0 will never work as a file and only crash, ever - could the file for that be archived or deleted from Curseforge?  Lots of people are still clicking update on the mod

**Comment by DarkS0ul-hope** [comment]
yeah same problem here on my modpack, please just archived the file it's gonna cause less trouble while you're working on the fix.

**Comment by memphismc** [comment]
can confirm, 4.0.20 also crashes on fabric 1.20.1

downgrading to 4.0.13 is the solution

[crash-2024-07-19_00.16.28-client.txt](https://github.com/user-attachments/files/16397263/crash-2024-07-19_00.16.28-client.txt)

**Comment by JT122406** [comment]
Fixed in newer versions and can be closed @CorgiTaco
